### DISCUSSION TITLE
Invalid pointer when using HistogramSplitter

### DISCUSSION THREAD

**Discussion by jpfeil** [online-ml/river#1556] [Q&A]
I've been getting these random failures when using the HisogramSplitter. I haven't been able to create a reproducible example, but I was wondering if anyone else has experienced this.

free(): invalid pointer                                                                                  
Aborted (core dumped)

**Comment by smastelini** [comment]
Hi @jpfeil, thanks for reporting that. What is the tree-based algorithm you are using along with the Histogram splitter?

**Comment by jpfeil** [comment]
I'm using the HoeffdingTreeClassifier. It randomly occurs during training, but I haven't been able to reproduce it. I'm also keeping track of a RollingAUC metric.

**Reply by smastelini** [reply]
This is really strange :/ The histogram splitter was not updated in a long time, so I doubt the problem is due to some recent change. Interestingly, the RollingAUC is implemented in C++, so the pointer stuff could come from there. But this is just a wild and non-informed guess. A reproducible example would be the first step to try and track the problem.
